infrastructure/README.md

```diff
@@ -0,0 +1,31 @@
+## Intent
+The goal of this deployment folder is to fully automate the infrastructure of the App Service Diagnostics application.
+This will make it trivial to create the resources needed to deploy ASD into new clouds.
+
+ARM templates also control Geneva configuration upgrades.
+
+#### Note
+Currently these deployments only work in the Azure Portal Powershell terminal
```

Comment:
These deployment doesn't work through Azure powershell on local machine?

Reply:
For some reason they fail to complete, even after getting portal access. The only way I could get them to deploy was by opening a portal powershell terminal.